src/main/resources/application.yml

```diff
@@ -1,2 +1,24 @@
 server:
-  port: 8080
\ No newline at end of file
+  port: 8080
+  shutdown: graceful
+
+spring:
+  lifecycle:
+    timeout-per-shutdown-phase: 30s
+
+management:
+  endpoint:
+    shutdown:
+      enabled: true
+  endpoints:
+    web:
+      exposure:
+        include:
+          - info
+          - health
+          - liquibase
+          - metrics
+          - shutdown
```

Comment:
- shutdown
          - prometheus
there is not necessary to close these endpoint with autentification? Or it will be made later?

Reply:
It will be done later